src/actions/searchOffersActions.spec.js

```diff
@@ -54,6 +55,17 @@ describe("Search Offers actions", () => {
         expect(setSearchValue(value)).toEqual(expectedAction);
     });
 
+    it("should return Set Show Hidden action", () => {
+
+        const showHidden = "show_hidden";
```

Comment:
Shouldn't this be true/false

Reply:
Now that I'm thinking maybe you are correct, because showHidden is a boolean, thank you I will take a look at it

Reply:
Done!